plugins/cilium-cni/cilium-cni.go

```diff
@@ -71,6 +71,14 @@ type netConf struct {
 	Args Args `json:"args"`
 }
 
+type cniArgsSpec struct {
+	cniTypes.CommonArgs
+	IP                         net.IP
+	K8S_POD_NAME               cniTypes.UnmarshallableString
```

Comment:
The change makes sense to me, but where are these arguments passed in when `cilium-cni` is invoked? Is this done via `kubelet?`